src/style.css

```diff
@@ -0,0 +1,27 @@
+* {
+  margin: 0;
+  padding: 0;
+}
```

Comment:
Do not apply any styles to the `*` selector except box-sizing

Reply:
done